content/community/courses.md

```diff
@@ -1,51 +1,57 @@
 ---
 id: courses
-title: Courses
+title: نصاب
 layout: community
 sectionid: community
 permalink: community/courses.html
 ---
 
-## Free Courses {#free-courses}
+## مفت نصاب {#free-courses}
 
-- [Glitch: React Starter Kit](https://glitch.com/glimmer/post/react-starter-kit/) - A free, 5-part video course with interactive code examples that will help you learn React.
+- [Glitch: React Starter Kit](https://glitch.com/glimmer/post/react-starter-kit/) - انٹرایکٹو کود مثالوں کے ساتھ بالکل مفت، ۵ ھصے کا ویڈیو کورس جو آپ کو ری ایکٹ(React) سیکھنے میں مدد فراہم کرے گا ۔
 
-- [Codecademy: React 101](https://www.codecademy.com/learn/react-101) - Codecademy's introductory course for React.
+- [Codecademy: React 101](https://www.codecademy.com/learn/react-101) - کوڈ اکیڈمی کا ری ایکٹ (React) کے لیے تعارفی کورس ۔
```

Comment:
Similarly you do not need to translate the names of other platforms i.e. Codeacademy

Reply:
Also do not need to wrap React into parentheses

Reply:
The problem is I'm having a syntax issue if I don't start with Urdu.
To overcome this I had to translate React and codeacademy in Urdu.

Reply:
![Screenshot 2020-08-08 at 3 14 13 PM](https://user-images.githubusercontent.com/30750191/89707675-e0979980-d989-11ea-8fe3-7f5321fffae4.png)

here you can see the issue.

Reply:
@jabranr ??

Reply:
That is something will be fixed when RTL is implemented correctly. For now you can wrap the line inside enforced RTL i.e.

`<span dir="rtl">YOUR TRANSLATION GOES HERE</span>`

Try this and see if that works for you.